endpoints_management/control/wsgi.py

```diff
@@ -284,9 +284,18 @@ def __call__(self, environ, start_response):
             check_resp = self._control_client.check(check_req)
             error_msg = self._handle_check_response(app_info, check_resp, start_response)
             if (check_resp and check_resp.checkInfo and
-                check_resp.checkInfo.consumerInfo):
+                    check_resp.checkInfo.consumerInfo):
                 consumer_project_number = (
                     check_resp.checkInfo.consumerInfo.projectNumber)
+            if error_msg is None:
+                quota_info = self._create_quota_info(method_info, parsed_uri, environ)
+                if not quota_info.quota_info:
+                    logger.debug(u'no metric costs for this method')
```

Comment:
Might be more helpful to log what method?

Reply:
probably; this is a direct translation of [the equivalent Java](https://github.com/cloudendpoints/endpoints-management-java/blob/711dc96d33ca0474ba832706e868c5f6300de991/endpoints-control/src/main/java/com/google/api/control/ControlFilter.java#L311) though.